src/TTimer.h

```diff
@@ -99,8 +103,23 @@ class TTimer : public Tree<TTimer>
     bool mNeedsToBeCompiled;
     QTimer* mpQTimer;
     bool mModuleMember;
-    // temporary timers are single-shot by default, unless repeating is set
-    bool mRepeating;
 };
 
+#ifndef QT_NO_DEBUG_STREAM
+inline QDebug& operator<<(QDebug& debug, const TTimer* timer)
+{
+    QDebugStateSaver saver(debug);
+    Q_UNUSED(saver);
+    debug.nospace() << "TTimer("
+                    << "name: " << timer->getName()
+                    << " time: " << timer->getTime()
+                    << " command: " << timer->getCommand()
+                    << " script is in: " << (timer->mRegisteredAnonymousLuaFunction ? "string": "Lua function")
+                    << " script: " << timer->getScript()
+                    << " repeating: " << timer->mRepeating
```

Comment:
Inconsistent code style - whilst I personally am a fan of `"introducer:" << value` this is not the same as the `"introducer="` << value` style in the other classes...